src/components/repository-profile.component.js

```diff
@@ -36,8 +36,8 @@ const styles = StyleSheet.create({
     color: colors.white,
     ...fonts.fontPrimary,
     fontSize: normalize(12),
-    paddingLeft: 15,
-    paddingRight: 15,
+    paddingLeft: 30,
+    paddingRight: 30,
```

Comment:
@junedomingo thanks, didn't know about that before. Used in d2ce96f

> Maybe a little bit of top padding?
For me currently look good, don't you like how the UI looks now?